website/client/js/controllers/inviteToGroupCtrl.js

```diff
@@ -44,8 +44,13 @@ habitrpg.controller('InviteToGroupCtrl', ['$scope', '$rootScope', 'User', 'Group
 
       Groups.Group.invite($scope.group._id, invitationDetails)
         .then(function() {
-          Notification.text(window.env.t('invitationsSent'));
-          _resetInvitees();
+           if ((inviteMethod === 'email' && invitationDetails.emails.length > 1) || (inviteMethod === 'uuid' && invitationDetails.uuids.length > 1)) { // test if it's inviting more than 1 emails or uuid
```

Comment:
@khipkin Are you sure that is correct? I made that change locally and the invite button stopped working.
I made those tests because if `inviteMethod === 'email'` then `invitationDetails.uuids` is not supposed to exist, same for `inviteMethod === 'uuid'` and `invitationDetails.emails`.